styles/typography.css

```diff
@@ -5,3 +5,65 @@
   --font-family-serif: "Source Serif 4", serif;
   --bs-font-sans-serif: "Source Sans 3", "Source Sans Pro";
 }
+
+h1, .h1 {
+  font-size: 2.25rem;
```

Comment:
I think we are supposed to keep the responsive text size for screens below 1200px, but I am not 100% sure about this.

Reply:
Ah – so all of this (except the parts that say "xs" and "xl", maybe?) would be for <= 1200px only?

Reply:
I think we are supposed to do both, i.e.
< 1200px responsive text sizes with the new variables (i.e. H1 gets calculated with 2.25rem not 2.5rem)
> 1200px fixed font where it is 2.25rem not 2.5rem

But honestly this ticket was kind of confusing to me. I think @jcoyne has a better handle than I do.

Reply:
Yes, it was my understanding that we would continue to have responsive text sizes.